docs/model_configuration.md

```diff
@@ -570,7 +570,7 @@ GPU 0 and two execution instances on GPUs 1 and 2.
 The instance group setting is also used to enable execution of a model
 on the CPU. A model can be executed on the CPU even if there is a GPU
 available in the system. The following places two execution instances
-on the CPU.
+(the default number of cpu instances) on the CPU.
```

Comment:
Does this documentation still make sense if it only applies to a specific allowlist of backends ? i.e. pytorch/openvino wouldn't have 2 instances by default if I understand core PR correctly?

Reply:
Maybe a statement somewhere on how "default number of cpu instances" will vary depending on the backend ?

Reply:
Updated the documentation explaining the opt in nature of the default cpu instance count.